### ISSUE TITLE
Passing options to filters

### ISSUE THREAD

**Issue by yann-morin-1998** [asciidoc-py/asciidoc-py2#40]
Hello!

It is impossible to pass options to external plugins:

```
["my-plugin",options="my-option"]
----
Text to be rendered.
----
```

It used to work in asciidoc 8.6.6, but is broken in asciidoc 8.6.9.

Such plugins that can have options are aafigure and ditaa:
    https://code.google.com/p/asciidoc-aafigure-filter/
    https://code.google.com/p/asciidoc-ditaa-filter/

For example, the aafigure filter documents this option:
    ["aafigure",options="proportional"]

For a complete reference, see this thread on the mailing list:
    https://groups.google.com/forum/#!topic/asciidoc/cs8-ihIKYXo

Thank you! :-)

**Comment by aerostitch** [comment]
Hi,

The python 2 implementation of asciidoc (which is what this repo is about) is deprecated.
Please move your issue to the python 3 implementation at https://github.com/asciidoc/asciidoc-py3 if still relevant on that newer implementation and close this issue.

Thanks for your help,
Joseph

**Comment by MasterOdin** [comment]
Fixed (in asciidoc/asciidoc-py3#120) 5.5 years later!
